frontend/apps/crates/components/src/module/_groups/cards/edit/sidebar/step_2/actions.rs

```diff
@@ -1,10 +1,10 @@
-use shared::domain::jig::module::body::{Background, ThemeChoice};
+use shared::domain::jig::module::body::{Background, ThemeId};
 
 use crate::module::_groups::cards::edit::state::*;
 
 impl<RawData: RawDataExt, E: ExtraExt> CardsBase<RawData, E> {
-    pub fn set_theme(&self, theme: ThemeChoice) {
-        self.theme_choice.set_neq(theme);
+    pub fn set_theme(&self, theme: ThemeId) {
```

Comment:
In #2337 I moved the separate implementations of `CardBase<> ` into "_groups/cards/edit/actions.rs" so that some of the functionality was accessible outside of individual steps - instead of having a super::actions, step_1::actions, etc, I merged them all together.